pyffs/backend.py

```diff
@@ -159,11 +161,11 @@ def ifftn(x, axes=None):
     mod : :obj:`func`
         Module to be used to process array (:mod:`numpy` or :mod:`cupy`)
     """
-
     if get_array_module(x) == np:
-        return scipy.fft.ifftn(x, axes=axes)
+        func = scipy.fft.ifftn
     else:
-        return cupyx.scipy.fft.ifftn(x, axes=axes)
+        func = cupyx.scipy.ifft.fftn
```

Comment:
@SepandKashani

Reply:
So it should be ``cupyx.scipy.fft.ifftn(x, axes=axes)`` based on the CuPy docs.